File: hospitalManagement/IHospitalActivities.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hospitalManagement
{
    internal interface IHospitalActivities
    {
        void AddFaculty();
        void AddRepository();
        void RemoveFaculty(string id);
        void RemoveRepository(string id);
        void ShowFaculty(string id);
        void ShowRepository(string id);
        void ShowAllFaculty();
        void ShowAllRepository();

        void UpdateFaculty(string id);
        void UpdateRepository(string id);
        void ClearFaculty();
        void ClearRepository();
    }
}

File: src/hospitalManagement/IHospitalActivities.java
package hospitalManagement;

public interface IHospitalActivities {
	void AddFaculty();
    void AddRepository();
    void RemoveFaculty(String id);
    void RemoveRepository(String id);
    void ShowFaculty(String id);
    void ShowRepository(String id);
    void ShowAllFaculty();
    void ShowAllRepository();
}
